src/test/kotlin/com/openlattice/hazelcast/serializers/SortedPrincipalSetStreamSerializerTest.kt

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Should be able to just have `SortedPrincipalSetStreamSerializer` implement `TestableSelfRegisteringStreamSerializer`